.travis.yml

```diff
@@ -3,12 +3,18 @@ jdk:
   - oraclejdk8
   - openjdk7
 
```

Comment:
Are you talking about lines above or below? Lines above were not changed. Lines below required to make tests run properly, but agree this could be done as a separate PR. Also it's also already properly handled in the update project structure branch along with the rest of required changes to Travis script.